examples/lib/defaultHtmlStepUi.js

```diff
@@ -81,35 +81,53 @@ function DefaultHtmlStepUi(_sequencer, options) {
         var inputDesc = isInput ? mapHtmlTypes(inputs[paramName]) : {};
         if (!isInput) {
           html += '<span class="output"></span>';
-        } else if (inputDesc.type.toLowerCase() == 'select') {
+        } 
+        else if (inputDesc.type.toLowerCase() == 'select') {
+
           html += '<select class="form-control target" name="' + paramName + '">';
           for (var option in inputDesc.values) {
             html += '<option>' + inputDesc.values[option] + '</option>';
           }
           html += '</select>';
-        } else {
+        }
+        else {
           let paramVal = step.options[paramName] || inputDesc.default;
-          html =
-            '<input class="form-control target" type="' +
-            inputDesc.type +
-            '" name="' +
-            paramName +
-            '" value="' +
-            paramVal +
-            '" placeholder ="' +
-            (inputDesc.placeholder || '');
-
-          if (inputDesc.type.toLowerCase() == 'range') {
-            html +=
-              '"min="' +
-              inputDesc.min +
-              '"max="' +
-              inputDesc.max +
-              '"step="' +
-              (inputDesc.step ? inputDesc.step : 1)+ '">' + '<span>' + paramVal + '</span>';
 
+          if(inputDesc.id == 'color-picker'){ // separate input field for color-picker
+            html+= 
+                '<div id="color-picker" class="input-group colorpicker-component">'+
+                 '<input class="form-control target" type="' +
+                  inputDesc.type +
+                  '" name="' +
+                  paramName +
+                  '" value="' +
+                  paramVal + '">'+ '<span class="input-group-addon"><i></i></span>' +
+                  '</div>';
+          }
+          else { // use this if the the field isn't color-picker
+            html =
+              '<input class="form-control target" type="' +
+              inputDesc.type +
+              '" name="' +
+              paramName +
+              '" value="' +
+              paramVal +
+              '" placeholder ="' +
+              (inputDesc.placeholder || '')+
+              '"id ="' + (inputDesc.id || '');
+
+            if (inputDesc.type.toLowerCase() == 'range') {
+              html +=
+                    '"min="' +
```

Comment:
You have added extra tab by mistake. Small fix.

Reply:
> You have added extra tab by mistake. Small fix.



yeahh I ll modify that, and also I have added indentation as another conditional level I have added.